lib/index.js

```diff
@@ -1,19 +1,25 @@
-"use strict";
+'use strict';
 
 const SteamCommunity = require('steamcommunity');
-const Helpers = require('./helpers.js');
-
-module.exports = TradeOfferManager;
-
-const SteamID = TradeOfferManager.SteamID = require('steamid');
-const ETradeOfferState = TradeOfferManager.ETradeOfferState = require('../resources/ETradeOfferState.js');
-const EOfferFilter = TradeOfferManager.EOfferFilter = require('../resources/EOfferFilter.js');
-const EResult = TradeOfferManager.EResult = require('../resources/EResult.js');
-const EConfirmationMethod = TradeOfferManager.EConfirmationMethod = require('../resources/EConfirmationMethod.js');
-const ETradeStatus = TradeOfferManager.ETradeStatus = require('../resources/ETradeStatus.js');
+const SteamID = require('steamid');
+const url = require('url');
+const languages = require('languages');
```

Comment:
I prefer to capitalize namespaces that come from `require`. This could be `URL` and `Languages`.